docs/advanced-features/custom-directories.md

```diff
@@ -0,0 +1,42 @@
+---
+description: Customize the directories included in your Next.js project.
```

Comment:
I'd name it "Customize the files included in your Next.js build"
- it's more generic, because we should also probably add the "collocated files + src/ pattern" to this documentation (in order to handle *.tests.js or *.stories.js alongside normal source code, in the same folder)
- it's clearer about what "included" means (=> included in the build)